correios/models/address.py

```diff
@@ -280,7 +281,8 @@ def __init__(
 
     @property
     def complement_safe_display(self) -> str:
-        return "".join(c for c in str(self.complement) if c.isalnum())
+        complement = unicodedata.normalize("NFKD", self.complement).encode("ascii", "ignore").decode()
+        return " ".join(complement.split())
```

Comment:
n entendi a necessidade desse join e split, pode me explicar?

Reply:
remover espaços duplicados..
no caso de teste com `15 • andar` na linha 284 o resultado é `15  andar`, pq ela só remove o caracter não-ascii, então aqui é um modo de remover esses espaços em branco a mais.. (funciona com outros tipos de whitespace tb..)

Reply:
agora entendi, será q teria como colocar um caso de teste mais explícito para isso? acho q ficaria mais fácil de ver/entender essa necessidade

Reply:
added a "many whitespaces" parameter in test_shipping_label_get_datamatrix_info_with_complement_display